Fixes the link to Administrative Safeguards PDF

madnesspie opened the pull request:
Hello! I've found that the link to Administrative Safeguards seems to be obsolete and fixed it. Thank you for your great work, thats really helpful!

jason-w reviewed (changes requested):
Thank you for the PR! Can you also update the URL text (in addition to the actual URL)? Once done, I'll merge the PR. Thanks!

madnesspie commented:
> Thank you for the PR! Can you also update the URL text (in addition to the actual URL)? Once done, I'll merge the PR. Thanks!

Sorry, I'm not sure that I understood you correctly. If you meant `[URL text]` then it has the same format as Technical Safeguards and Physical Safeguards and I don't know what's wrong with it. If you meant the chrome highlight syntax `#:~:text=...` so I added it and removed redirects.
